src/app/data-planes/data/index.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  DataPlaneOverview,
+} from '@/types/index.d'
+
+export function parseMTLSData(dataPlaneOverview: DataPlaneOverview) {
+  if (dataPlaneOverview.dataplaneInsight === undefined || dataPlaneOverview.dataplaneInsight.mTLS === undefined) {
+    return null
+  }
+
+  const rawExpDate = new Date(dataPlaneOverview.dataplaneInsight.mTLS.certificateExpirationTime)
+  // this prevents any weird date shifting
+  const fixedExpDate = new Date(rawExpDate.getTime() + rawExpDate.getTimezoneOffset() * 60000)
+
+  return {
+    ...dataPlaneOverview.dataplaneInsight.mTLS,
+    certificateExpirationTime: fixedExpDate.toISOString(),
+  }
+}
```

Comment:
This was moved out of `utilities/dataplane.ts`, but I also removed the formatting that it used to have here and moved that formatting to the template instead.